docs/cli/interactive-manual-testing.md

```
@@ -0,0 +1,178 @@
+### 1.1 Sin argumentos
+
+```bash
+cd tools/cli
+npm run start:ts
+```
+(La primera vez ejecuta `npm install` para instalar `tsx`.)
+
+**Evidencia:** Captura donde se vea:
+- Mensaje de bienvenida: "Galaxy DevKit Interactive Mode"
+- "Type 'help' for commands or 'exit' to quit"
+- Prompt `galaxy> `
```

markdownlint-cli2 (0.20.0)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)